package com.silver.diary.entity;

import com.baomidou.mybatisplus.annotation.*;
import lombok.Data;

@Data
@TableName("article_like")
public class ArticleLike {
    @TableId(type = IdType.AUTO)
    private Integer id;

    private Integer articleId;
    private Integer userId;
}
